space-concordia-robotics/robotics-prototype
# Fix Local IK Launch File and README

# Assignee Section

## Description
- Fix readme (add missing steps)
- Make launchfile for local ik work
### Steps for Testing
1. Run local_ik.launch.py
2. Plug in a controller
3. See the controller position update in RViz

The approval from all software team leads is necessary before merging.

# Reviewer Section

Aside from local testing and the General Integration Test it is implied that static analysis should be included in the verification process.


For Pull Requests that do not include code changes, it is not required to perform the tests above.
